GetFed/GetFed/Helpers/CustomTextField.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  CustomTextField.swift
+//  GetFed
+//
+//  Created by Britney Smith on 12/7/18.
+//  Copyright © 2018 Britney Smith. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class CustomTextField: UITextField {
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        underline()
+    }
+    
+    func underline() {
+        let border = CALayer()
+        let width = CGFloat(1.25)
+        guard let customBorderColor = UIColor(named: ColorID.AccentColorB.rawValue) else {
+            print("No border color!")
```

Comment:
:cake: Replace print statement with something else (maybe using a default border color? though I suppose that would happen anyway 🤔 )

Reply:
@leeanndrees Agreed! I've changed this line to `let customBorderColor = UIColor(named: ColorID.AccentColorB.rawValue) ?? UIColor.black`

Reply:
Yay! Love it!